base/db/R/query.dplyr.R

```diff
@@ -235,6 +234,72 @@ var_names_all <- function(bety, workflow_id, run_id) {
 #' @param run_id Run ID
 #' @param workflow_id Workflow ID
 #' @export
+# load_data_single_run <- function(bety, workflow_id, run_id) {
+#   # For a particular combination of workflow and run id, loads
+#   # all variables from all files.
+#   # @return Dataframe for one run
+#   # Adapted from earlier code in pecan/shiny/workflowPlot/server.R
+#   globalDF <- data.frame()
+#   workflow <- dplyr::collect(workflow(bety, workflow_id))
+#   # Use the function 'var_names_all' to get all variables
+#   var_names <- var_names_all(bety, workflow_id, run_id)
+#   # TODO: This looks a lot like `read.output`. Should probably just use that here.
+#   # Using earlier code, refactored
+#   if (nrow(workflow) > 0) {
+#     outputfolder <- file.path(workflow$folder, 'out', run_id)
+#     files <- list.files(outputfolder, "*.nc$", full.names = TRUE)
+#     for (file in files) {
+#       nc <- nc_open(file)
+#       for (var_name in var_names) {
+#         dates <- NA
+#         vals <- NA
+#         title <- var_name
+#         ylab <- ""
+#         var <- ncdf4::ncatt_get(nc, var_name)
+#         #sw <- if ('Swdown' %in% names(nc$var)) ncdf4::ncvar_get(nc, 'Swdown') else TRUE
+#         # Snow water
+#         sw <- TRUE
+#         # Check required bcoz many files don't contain title
+#         if (!is.null(var$long_name)) {
+#           title <- var$long_name
+#         }
+#         # Check required bcoz many files don't contain units
+#         if (!is.null(var$units)) {
+#           ylab <- var$units
+#         }
+#         x <- ncdays2date(ncdf4::ncvar_get(nc, 'time'), ncdf4::ncatt_get(nc, 'time'))
+#         y <- ncdf4::ncvar_get(nc, var_name)
+#         b <- !is.na(x) & !is.na(y) & sw != 0
+#         
+#         dates <- if(is.na(dates)) x[b] else c(dates, x[b])
+#         dates <- as.POSIXct(dates)
+#         vals <- if(is.na(vals)) y[b] else c(vals, y[b])
+# 
+#         xlab <- "Time"
+#         # Values of the data which we will plot
+#         valuesDF <- data.frame(dates,vals)
+#         # Meta information about the data.
+#         metaDF <- data.frame(workflow_id,run_id,title,xlab,ylab,var_name)
+#         # Meta and Values DF created differently because they would of different
+#         # number of rows. cbind would repeat metaDF(1X6) to the size of valuesDF
+#         currentDF <- cbind(valuesDF,metaDF)
+#         globalDF <- rbind(globalDF,currentDF)
+#       }
+#       ncdf4::nc_close(nc)
+#     }
+#   }
+#   # Required to convert from factors to characters
+#   # Otherwise error by ggplotly
+#   globalDF$title <- as.character(globalDF$title)
+#   globalDF$xlab <- as.character(globalDF$xlab)
+#   globalDF$ylab <- as.character(globalDF$ylab)
+#   globalDF$var_name <- as.character(globalDF$var_name)
+#   return(globalDF)
+# } #load_data_single_run
+
+# Currently commenting out previous version of load_data_single_run
+# The following version ususe read.output instead of alternative code 
+
```

Comment:
If the below code works, can you just delete this? It's safely stored in our git history if we ever need it.

Reply:
@ashiklom do you mean delete all of load_data_single_run or @bcow's globalDF changes?

Reply:
I just mean delete the commented-out old version of `load_data_single_run` function so it isn't taking up space.